components/navbar/navbarPage.css

```diff
@@ -0,0 +1,20 @@
+.container__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container__content__sub-title {
+    align-self: flex-start;
+    margin-top: 3rem;
+    margin-bottom: 2rem;
+}
+
+.container__code-preview {
+    width: 80rem;
+    min-height: 22rem;
+    margin-top: 5rem;
+    align-self: center;
+    justify-content: center;
+    overflow: scroll;
```

Comment:
Add auto from now so it would be easy for future

Reply:
Done.